packages/cli/lib/utils.js

```diff
@@ -5,14 +5,21 @@
 
 'use strict';
 
+// TODO(kjdelisle): Drop this shim once we drop Node 6.
+require('util.promisify/shim')();
```

Comment:
So AFAICT, `util.promisify/shim.js` is modifying Node.js built-in `util` export - that's not good, but easy to fix by using `util.promisify/implementation` instead. It's also trying to support `kCustomPromisifiedSymbol`, but I don't think that will work unless the shim is loaded as the first thing in the app, so that 3rd party modules have access to this symbol. I guess that does not matter much since we are going to drop support for Node.js 6.x soon anyways.

As I am thinking about this, I guess as long as you use `implementation` instead of `shim`, so that we don't modify built-in `require('util')` then I am fine with your change. Please consider updating the existing `promisify` function in `core` to use the same implementation under the hood, so that we have consistent behaviour. Preferably as part of your PR (see also mine https://github.com/strongloop/loopback-next/pull/848/files#diff-9dab89a9fdf9229c03e541d55747f3bf)